File: 1078 - Otro dulces/problem_1078.java
import java.util.Scanner;

public class problem_1078 {
    public static void main(String[] args) {
        Scanner e = new Scanner(System.in);
        int T = e.nextInt();
        while(true)
        {
            if(T==0)
                break;
            int num = e.nextInt();
            int sum = 0;
            
            for(int i = 0; i<num; i++)
            {
                sum += e.nextInt();
            }
            if(sum%num==0)
                System.out.println("YES");
            else
                System.out.println("NO");
            T--;
        }
    }
}

File: 1078 - Otro dulces/problem_1078.cpp
#include <bits/stdc++.h>
using namespace std;


int main(){
	int T, num;
	std::cin >> T;
	while(true)
	{
		if(T==0)
			break;
		int NN, suma=0;
		std::cin >> NN;
		for(int i=0; i<NN; i++)
		{
			std::cin >> num;
			suma += num;
		}
		if(suma%NN==0)
			printf("YES\n");
		else
			printf("NO\n");
		T--;	
	}		
	return 0;
}
